de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/InstantIncomingReferencePointNegotiation.java

```diff
@@ -32,16 +32,16 @@
  * Receive shared Projects and display them instant using a stream based
  * solution.
  */
-public class InstantIncomingProjectNegotiation extends
-    AbstractIncomingProjectNegotiation {
+public class InstantIncomingReferencePointNegotiation extends
+    AbstractIncomingReferencePointNegotiation {
 
     private static final Logger log = Logger
-        .getLogger(InstantIncomingProjectNegotiation.class);
+        .getLogger(InstantIncomingReferencePointNegotiation.class);
 
-    public InstantIncomingProjectNegotiation(
+    public InstantIncomingReferencePointNegotiation(
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 11 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMe8DZywvPe4LUfW&open=AWZfKMe8DZywvPe4LUfW)

[ISSUE_KEY]: AWZfKMe8DZywvPe4LUfW